src/validators/idNumber.js

```diff
@@ -1,36 +1,82 @@
 'use strict'
 
-var idValidate = require('./south-african-id-number-validator').validate
+var moment = require('moment')
 
 module.exports = function ($compile) {
-
   return {
-    restrict: "A",
-    require: "ngModel",
+    restrict: 'A',
+    require: 'ngModel',
     scope: {
       checkIdNumber: '=checkIdNumber'
     },
-    link: function(scope, element, attributes, ngModel) {
-
-      scope.$watch('checkIdNumber', function(newValue, oldValue) {
+    link: function (scope, element, attributes, ngModel) {
+      scope.$watch('checkIdNumber', function (newValue, oldValue) {
```

Comment:
Maybe this should be `checkRSAIdNumber` as it's very specific to RSA IDs? Are we able to write more validators and plug them in easily for other IDs?

Reply:
I see there it still the SA validator below. I'm not sure I understand the difference between the two. Could you explain?

Reply:
The file can be removed as this was the old location for the RSA validate function.